### PR TITLE
[Feat: #152188237] Centralize Logging

### PR DESCRIPTION
**Pull request by JamesNyamu**
- add deployment tag to log object

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,19 +5,22 @@ const BugsnagTransport = require('winston-bugsnag').BugsnagTransport;
 const env = process.env.NODE_ENV || 'development';
 const VError = require('verror').VError;
 
-const podName = process.env.POD_NAME || 'myService-1';
-const values = podName.split('-');
+const podName = process.env.POD_NAME || 'default-service-1234567890-abcde';
+const deploymentTag = process.env.DEPLOYMENT_TAG || 'abcde'
+const serviceName = podName.substring(0, podName.length - 17); // Pod name : servicename +  '-' + 10 bytes numeric + '-' + 5 bytes alphanumeric string
 const serviceContext = {
```
Comment (andela-anandaa) [inline index.js:11]: How is this `serviceContext` passed to the logger?

Comment (JamesNyamu) [reply index.js:11]: @andela-anandaa Check out line 28 in the file.

Comment (andela-anandaa) [reply index.js:11]: @JamesNyamu - good stuff! Thanks. Solid work 👍

**Review by andela-anandaa — APPROVED** [review]

**Review by codesword — APPROVED** [review]
LGTM

**Pull request merged**